### PR TITLE
istat-menus: Remove U+2069 (Unicode POP DIRECTIONAL ISOLATE) from LaunchDaemons path

### PR DESCRIPTION
**Pull request by jivanpal**
Cask file for `istat-menus` currently has a couple of unnecessary Unicode PDI characters, i.e. the following is shown in `less`:

```
51          "/Library<U+2069>/LaunchDaemons<U+2069>/com.bjango.istatmenus.fans.plist",
```

This pull request removes these characters.

***

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask** ... (N/A)

### CODE REVIEW

File: Casks/istat-menus.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "~/Library/Preferences/com.bjango.istatmenus#{version.major}.extras.plist",
     "~/Library/Preferences/com.bjango.istatmenus.status.plist",
     "/Library/Logs/iStat Menus",
-    "/Library⁩/LaunchDaemons⁩/com.bjango.istatmenus.fans.plist",
+    "/Library/LaunchDaemons/com.bjango.istatmenus.fans.plist",
```
Comment (miccal) [inline Casks/istat-menus.rb:51]: Please move these two items, namely:
```
"/Library/Logs/iStat Menus",
"/Library/LaunchDaemons/com.bjango.istatmenus.fans.plist",
```
to `uninstall delete:`, as they are not user-specific files.

Comment (jivanpal) [reply Casks/istat-menus.rb:51]: @miccal, should this also be done with `/Users/Shared/.iStatMenus` on line 52? I am not familiar with the conventions of Cask files. You should have edit privileges, so feel free to make any other changes that you see fit.

Comment (miccal) [reply Casks/istat-menus.rb]: No, I think `/Users/Shared/.iStatMenus` is fine in the `zap` stanza.

**Comment by miccal** [comment]
Thank you @jivanpal.

**Review by miccal — APPROVED** [review]

**Pull request merged**